src/parameters.jl

```diff
@@ -47,6 +47,42 @@ function flatten(x::Positive)
     return v, unflatten_Positive
 end
 
+"""
+    Bounded{T<:Real, V<:Bijector, Tε<:Real}
```

Comment:
Maybe change this to the signature of the method that you want people to call. So something like
```julia
Bounded(value::Real, lower_bound::Real, upper_bound::Real)
```